oms-azurestorageanalytics-solution/azuredeploy.parameters.json

```diff
@@ -8,6 +8,9 @@
     "omsLogAnalyticsRegion": {
       "value": "eastus"
     },
+    "omsLogAnalyticsSku": {
```

Comment:
using "free" sku as the default value is causing the deployment to fail in this subscription. Please use one of the other (paid) sku's